CHANGELOG.md

```diff
@@ -12,18 +12,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Added
-- Add quickstart guide for background blur feature.
+
+- [Doc] Add quickstart guide for background blur feature.
```

Comment:
Don't need CHANGELOG for this.

Reply:
Got it, do you want me to also delete "Add quickstart guide for background blur feature."

Reply:
Removed all doc related CHANGELOG of this release.